src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -171,11 +159,7 @@ var adyenHelperObj = {
     };
   },
 
-  getAdyenRecurringEnabled() {
-    return adyenHelperObj.getCustomPreference('AdyenRecurringEnabled');
-  },
-
-  getAdyenOneClickEnabled() {
```

Comment:
Should we keep the "getAdyenOneClickEnabled" and delete the "getAdyenRecurringPaymentsEnabled" instead? This way it is more consistent with the configuration as the name in the Custom Preference is `AdyenOneClickEnabled`. What do you think?